Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/StepDefinitions/FilesStepDefinitions.cs

```diff
@@ -68,5 +86,105 @@ public async Task WhenISendsAGetRequestToTheFilesIdEndpoint(string identityName,
         _responseContext.WhenResponse = await identity.Files.GetFileMetadata(file.Id);
     }
 
+    [When($"{RegexFor.SINGLE_THING} sends a PATCH request to the /Files/{RegexFor.SINGLE_THING}.Id/ClaimFileOwnership with token {RegexFor.SINGLE_THING}.OwnershipToken")]
+    public async Task WhenISendsAPatchRequestToTheFilesFIdClaimFileOwnershipFOwnershipTokenEndpoint(string identityName, string fileName, string fileName2)
+    {
+        var identity = _clientPool.FirstForIdentityName(identityName);
+        var fileId = _filesContext.Files.FirstOrDefault(f => f.Key == fileName).Value?.Id ?? fileName;
+        var token = _filesContext.FileNameToOwnershipToken.FirstOrDefault(t => t.Key == fileName2).Value ?? "NonExistingTokenXXXX";
```

Comment:
Instead of using a default value for the ownership token, you should probably rather create a separate step definition. Just like in the comment above, I think it's confusing as it currently is.